src/components/MenuItem.js

```diff
@@ -364,6 +365,11 @@ const MenuItem = React.forwardRef((props, ref) => {
                                         />
                                     </View>
                                 )}
+                                {Boolean(props.rightLabel) && (
+                                    <View style={[styles.popoverMenuIcon, styles.rightLabelMenuItem, styles.pointerEventsAuto]}>
```

Comment:
```suggestion
                                    <View style={[styles.justifyContentCenter, styles.pointerEventsAuto]}>
```

Reply:
Also, what's `styles.pointerEventsAuto` for? I don't see any difference with/without this style.